guides/appendix/common-issues.md

```diff
@@ -34,7 +27,7 @@ git config --global url."https://".insteadOf git://
 
 In most cases you should use a stable release, but if you need to install a canary version to test beta features, you'd do it like this:
 
-For Ember: `bower install ember#canary --resolution canary`
+For Ember: `npm install --save-dev https://s3.amazonaws.com/builds.emberjs.com/canary/shas/106bb03e87f982657edd4595652579bfeae3f06a.tgz`
```

Comment:
I don't think we should list a specific canary SHA here like this. Instead, can you update the prose to something like:

```md
In most cases you should use a stable release, but if you need to 
install a canary version to test some new feature you can update
the `ember-source` entry in your `package.json` to use the latest
canary tarball URL. You can find the most recent canary tarball URL
by way of the 
[ember-source-channel-url](https://github.com/ember-cli/ember-source-channel-url)
package.

That might look like:

`npx ember-source-channel-url canary --write`
```

Reply:
fixed @rwjblue